unit_tests/test_ycsb_thread.py

```diff
@@ -28,9 +28,18 @@
 ALTERNATOR_PORT = 8000
 TEST_PARAMS = dict(dynamodb_primarykey_type='HASH_AND_RANGE',
                    alternator_use_dns_routing=True, alternator_port=ALTERNATOR_PORT)
-ALTERNATOR = alternator.api.Alternator(sct_params={"alternator_access_key_id": None,
-                                                   "alternator_secret_access_key": None,
-                                                   "alternator_port": ALTERNATOR_PORT})
+USE_ALTERNATOR_TTL = False  # Set 'True' for using TTL
+ALTERNATOR_WITH_TTL = alternator.api.Alternator(sct_params={"alternator_access_key_id": None,
+                                                            "alternator_secret_access_key": None,
+                                                            "alternator_port": ALTERNATOR_PORT,
+                                                            "experimental": "true",
+                                                            "append_scylla_yaml": " | "
```

Comment:
I don't think you need a pipe here, this is not a yaml, `append_scylla_yaml` is just a multi-line string

Reply:
also that won't really help in this test, since scylla isn't getting configured with this parameter,
in this unit test we run docker, see conftest.py for the details.

Reply:
removed related test. no need testing for only a single parameter.